geostore/check_stac_metadata/utils.py

```diff
@@ -138,15 +147,41 @@ def run(self, metadata_url: str) -> None:
                 LOG_MESSAGE_VALIDATION_COMPLETE,
                 extra={
                     "outcome": Outcome.FAILED,
-                    "error": NO_ASSETS_FOUND_ERROR_MESSAGE,
+                    "error": Check.NO_ASSETS_IN_DATASET,
                     GIT_COMMIT: get_param(ParameterName.GIT_COMMIT),
                 },
             )
             return
 
+        stac_type = self.get_stac_type_by_url(metadata_url)
+        if not is_instance_of_catalog_or_collection(stac_type):
+            error_message = (
+                f"Uploaded Assets should be catalog.json or collection.json”: “{metadata_url}”"
```

Comment:
The first "”" probably shouldn't be there.

Reply:
Thanks for spotting that. Updated